ProxyAgent/Properties/launchSettings.json

```diff
@@ -12,19 +12,22 @@
       "commandName": "IISExpress",
       "launchBrowser": true,
       "environmentVariables": {
-        "REMOTE_ENDPOINT": "http://vm-snjz6.westus.cloudapp.azure.com",
-        "Key": "Value",
+        "REMOTE_ENDPOINT": "https://vm-123456.somecloud.cloudapp.azure.com",
+        "REDIRECT_HTTP_TO_HTTPS": "false",
+        "REMOTE_ENDPOINT_SSL_THUMBPRINT": "0011223344556677889900aabbccddeeff001122",
         "ASPNETCORE_ENVIRONMENT": "Development"
       }
     },
     "ProxyAgent": {
       "commandName": "Project",
-      "launchBrowser": false,
       "launchUrl": "status",
       "environmentVariables": {
+        "REMOTE_ENDPOINT": "https://vm-123456.somecloud.cloudapp.azure.com",
+        "REDIRECT_HTTP_TO_HTTPS": "false",
```

Comment:
In the README, the default for REDIRECT_HTTP_TO_HTTPS value is true, should this be true also?

Reply:
It makes development easier if the proxy doesn't redirect by default, allowing to test the proxying feature. One can test the redirection feature changing this value of course, but the proxy will not run over SSL, so the traffic won't go through.